examples/jet/pages/fullscreen.tsx

```diff
@@ -0,0 +1,132 @@
+    colors: {
+      bg: 'bg-[#E5EBF4]',
+      errorBg: 'bg-[#DC726D]',
+      primary: 'text-[#444444]',
+      secondary: 'text-[#E6EBF3]',
+      accent: 'text-gradient',
+      accentSolid: 'text-[#5895B9]',
+      brand: 'text-[#E5EBF4]',
+    },
+    textStyles: {
+      h1: 'font-poppins text-xl font-normal',
+      body: 'font-sans text-sm leading-snug ',
+      small: 'font-sans text-[13px] leading-snug',
+      bigText: 'text-base font-medium leading-snug',
+      header: 'font-poppins text-lg',
+      buttonText: 'text-xs uppercase tracking-[1.5px] text-[#E6EBF3]',
+      bigButtonText: 'text-xs uppercase tracking-[1.5px]',
+      bigButtonSubtle: 'text-xs uppercase tracking-[1.5px]',
+    },
+    icons: {
+      bell: BellIcon,
+      back: BackIcon,
+      settings: SettingsIcon,
+    },
+    bellButton: 'w-10 h-10 jet-shadow-light',
+    header: 'px-2 py-4',
+    popup: 'rounded-3xl jet-shadow',
+    button: 'bg-gradient text-[#E6EBF3] hover:opacity-60',
+    buttonLoading: 'bg-gradient min-h-[32px] opacity-20',
+    bigButton: 'text-white hover:opacity-60',
+    bigButtonLoading: 'text-white min-h-[32px] opacity-20',
+    divider: 'h-[4px] rounded-lg bg-gradient-to-b from-[#C3CADE] to-[#F8F9FB]',
+    highlighted: 'px-2 py-1 rounded-lg bg-white/30',
+  },
+  dark: {
+    colors: {
+      bg: 'bg-[#444444]',
+      errorBg: 'bg-[#DC726D]',
+      primary: 'text-white',
+      secondary: 'text-[#E6EBF3]',
+      accent: 'text-gradient',
+      accentSolid: 'text-[#5895B9]',
+      brand: 'text-[#E5EBF4]',
+    },
+    textStyles: {
+      h1: 'font-poppins text-xl font-normal',
+      body: 'font-sans text-sm leading-snug ',
+      small: 'font-sans text-[13px] leading-snug',
+      bigText: 'text-base font-medium leading-snug',
+      header: 'font-poppins text-lg',
+      buttonText: 'text-xs uppercase tracking-[1.5px] text-[#444]',
+      bigButtonText: 'text-xs uppercase tracking-[1.5px]',
+      bigButtonSubtle: 'text-xs uppercase tracking-[1.5px]',
+    },
+    icons: {
+      bell: BellIcon,
+      back: BackIcon,
+      settings: SettingsIcon,
+    },
+    bellButton: 'w-10 h-10 jet-shadow-dark',
+    header: 'px-2 py-4',
+    popup: 'rounded-3xl jet-shadow',
+    button: 'bg-gradient text-[#E6EBF3] hover:opacity-60',
+    buttonLoading: 'bg-gradient min-h-[32px] opacity-20',
+    bigButton: 'text-white hover:opacity-60',
+    bigButtonLoading: 'text-white min-h-[32px] opacity-20',
+    divider: 'h-[4px] rounded-lg bg-gradient-to-b from-[#3C3C3C] to-[#505050]',
+    highlighted: 'px-2 py-1 rounded-lg bg-[#ABABAB]/10',
+  },
+};
+
+function HomeWithWallet() {
+  const wallet = useWallet();
+  const [theme, setTheme] = useState<'dark' | 'light'>('light');
+
+  const toggleTheme = useCallback(() => {
+    setTheme(theme === 'dark' ? 'light' : 'dark');
+  }, [theme]);
+
+  return (
+    <>
+      <DialectProvider
+        wallet={wallet}
+        network={'devnet'}
+        publicKey={DIALECT_PUBLIC_KEY}
+        theme={theme}
```

Comment:
not specific to this pr but wondering if there's a better way to organize darkmode/lightmode (`theme`) together with the injected styles for those modes (`variables={themeVariables}`). it wasn't immediately clear what each was or why they're different from each other

Reply:
good feedback! One solution could be to rename the `variables` into `themes` and `theme` to `currentTheme` so it will be clear what you need to choose from what in variables dictionary ("light"/"dark")